### PR TITLE
Feature/fix sequence number defaulting in image receipt

### PR DESCRIPTION
**Pull request by tundish**
This fixes the following issue seen in MWSS survey:
Image receipt files (names of the form EDC_134_YYYYMMDD_XXXX.csv where XXXX is a sequence number) were being created with sequence number stuck at 1000.

This was because MWSSTransformer failed to pass the sequence number to ImageTransformer.
ImageTransformer defaults to 1000.

### CODE REVIEW

File: tests/test_mwss_transform.py [hunk 1]
```diff
@@ -579,3 +581,25 @@ def test_tempdir(self):
             tfr.pack(img_seq=itertools.count())
         except KeyError:
             self.fail("TODO: define pages of survey.")
+
+    def test_image_sequence_number(self):
```
Comment (JamesGardiner) [inline tests/test_mwss_transform.py]: Can we add another unit test that covers the no `seq_nr` scenario to make sure the error deosn't make its way downstream?

File: transform/transformers/MWSSTransformer.py [hunk 2]
```diff
@@ -437,7 +437,7 @@ def pack(self, img_seq=None):
             doc.build(PDFTransformer.get_elements(survey, self.response))
 
             # Create page images from PDF
-            img_tfr = ImageTransformer(self.log, survey, self.response)
+            img_tfr = ImageTransformer(self.log, survey, self.response, self.ids.seq_nr)
```
Comment (JamesGardiner) [inline transform/transformers/MWSSTransformer.py]: Happy to approve once we show that defaulting to 0 in `seq_nr` doesn't cause further issues downstream via unit tests

**Comment by tundish** [comment]
Raised #64 by request.

**Comment by eldeal** [comment]
Please update changelog with reference to issue ( as in https://github.com/ONSdigital/sdx-transform-cs/blob/develop/CHANGELOG.md#130-2016-11-23 ) and make sure branch is up to date with develop. Then ready to merge

**Pull request merged**